### DISCUSSION TITLE
How can I fix this serious git clone error?

### DISCUSSION THREAD

**Discussion by eduardz097** [community/community#48568] [Programming Help] [answered]
### Select Topic Area

Question

### Body

I tried the following:
```
E:\Workspace\Blockchain>git clone https://github.com/substrate-developer-hub/substrate-node-template.git
Cloning into 'substrate-node-template'...
remote: Enumerating objects: 2185, done.
remote: Counting objects: 100% (91/91), done.
remote: Compressing objects: 100% (55/55), done.
Receiving objects: 94%(2062/2185), 3.18 MiB | 9.00 KiB/s
```
After a 20 minute delay at 94%, the result is:
```
E:\Workspace\Blockchain>git clone https://github.com/substrate-developer-hub/substrate-node-template.git
Cloning into 'substrate-node-template'...
remote: Enumerating objects: 2185, done.
remote: Counting objects: 100% (91/91), done.
remote: Compressing objects: 100% (55/55), done.
error: 6546 bytes of body are still expectediB | 11.00 KiB/s
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
```
How can I fix it?

**Comment by valentinojosh** [comment, marked as answer]
Hi!

There could be a lot of different things going on so I'm going to give you a general list and hopefully something helps!

- If you're using a VPN, disconnect from it
- Restart router and computer
- Try on a different network
- Try from a different workspace
- Try: `git clone --depth=1 https://github.com/substrate-developer-hub/substrate-node-template.git`

If none of those steps work you might want to consider downloading the repo as a zip and working with it locally. 

Hope this helps!

**Reply by eduardz097** [reply]
Hi, Josh.
Thanks a lot.
it works for me.

**Reply by Momellouky** [reply]
Hello Valentinojosh 
The last one worked perfectly in my case. Thank you.

**Reply by saretoduncan** [reply]
Hi, it worked for me. Thank you

**Reply by linkhl09** [reply]
Thanks!

**Reply by Merry9** [reply]
The last one worked perfectly in my case. Thank you very much!

**Reply by Blackhawk8899** [reply]
Hi i was facing same difficulty but it worked for me thanks

**Reply by milanmanvar-einfochips** [reply]
--depth=1 is hero

**Reply by umer-814** [reply]
i have a same problem in AndroRat ...
please tell me i can i do to solve this problem ??

**Reply by Tuvshu247** [reply]
ty

**Reply by nirm2009** [reply]
--depth=1 is hero (2)
For anyone wondering, --depth means the number of commits to grab when you clone.
[https://stackoverflow.com/questions/53683896/what-does-depth-for-git-clone-mean](https://stackoverflow.com/questions/53683896/what-does-depth-for-git-clone-mean)

**Reply by Princeizekor** [reply]
Thank you very much for this solution:

--depth=1 is hero

**Reply by Nazmul6180** [reply]
I tried all of this from a stack reply 

>

`` First, turn off compression

`git config --global core.compression 0`
Next, let's do a partial clone to truncate the amount of info coming down:

`git clone --depth 1 <repo_URI>`
When that works, go into the new directory and retrieve the rest of the clone:

`git fetch --unshallow `
or, alternately,

`git fetch --depth=2147483647`
Now, do a regular pull:

`git pull --all`
I think there is a glitch with msysgit in the 1.8.x versions that exacerbates these symptoms, so another option is to try with an earlier version of git (<= 1.8.3, I think). "

And all you have mentioned . 
But none of them worked 😔
**Actually connecting to a VPN was the solution for me.**

**Reply by khraoverflow** [reply]
this fixed it for some reason,  it was working fine but today refused to and this helped:
git config --global http.postBuffer 157286400 

ref: https://stackoverflow.com/questions/59282476/error-rpc-failed-curl-92-http-2-stream-0-was-not-closed-cleanly-protocol-erro

**Reply by nirm2009** [reply]
The cause can surely end up being a network issue, as well. 

Encountered the same issue again and *--depth=1 fix* didn't work this time. Retried 2-3 times, disconnected and connected to the WiFi, same issue. Just to try my luck, turned on a VPN and ironically that made the pull finally work.

**Reply by Chakit22** [reply]
Disconnecting from WIFI and connecting to a mobile data worked for me. I think because mobile network hasn't any VPN enabled that's why it worked. Correct me if I am wrong @valentinojosh ? Connecting my system with the mobile data (in this case the VPN is not enabled right).

**Reply by pranay-dutta** [reply]
what is the meaning of --depth=1 flag

**Reply by unfurkan** [reply]
Without being %100 sure  --depth=1  fetches only the latest commit along with its associated files, without retrieving the complete commit history

**Reply by junior3199** [reply]
In my case, it works by changing network connection from WiFi to mobile data. Still couldn't figure out the problems tho.

**Reply by brandenvs** [reply]
Truly hilarious.

**Reply by raggi** [reply]
It's not a network issue, it's a bug in the now extremely stale windows ssh release that has been fixed but they've failed to ship a production release:

https://github.com/PowerShell/Win32-OpenSSH/issues/1322

The fix has been in for nearly a year now but has been missed in every update.

You can update to their beta with `winget install Microsoft.OpenSSH.Beta`

**Reply by isaacobenzy** [reply]
Having same issue and i have restart the machine like 6 times today

**Reply by souvik666** [reply]
> Having same issue and i have restart the machine like 6 times today
> 
> 

Did you found a solution

**Reply by venkateshkolisetty-ctl** [reply]
Worked for me after disconnecting VPN. Thank you.

**Reply by Success-K** [reply]
I tried a different network and it worked for me.

**Reply by TroutZhang** [reply]
@raggi Unfortunately, it's not totally fixed yet.    Maybe it decreases the chance of failure.
Check my latest comments here https://github.com/PowerShell/Win32-OpenSSH/issues/2012 

Update 1w later:
It's **fixed**, I didn't make sure my git is using the newer version of SSH.exe, check https://github.com/PowerShell/Win32-OpenSSH/issues/2012#issuecomment-2242149102

**Reply by Bilalsheikh14** [reply]
The error you're encountering, "error: 6546 bytes of body are still expected," and "fetch-pack: unexpected disconnect while reading sideband packet," typically indicates an issue with the network connection or the remote repository's availability.

Here are a few steps you can try to resolve the issue:

Check your internet connection: Make sure you have a stable internet connection. Try running a speed test or accessing other websites to verify that your connection is working properly.

Retry the clone operation: Sometimes, a temporary network glitch or server issue can cause the clone operation to fail. Try running the git clone command again.

Use a different protocol: Instead of using the HTTPS protocol (https://github.com/...), try cloning the repository using the SSH protocol ([git@github.com](mailto:git@github.com):...). This can help bypass potential issues with HTTPS connections.

Clear Git's cache: Git stores a cache of remote repository data, which can sometimes become corrupted. Try clearing the cache by running the following command and then retrying the clone:

bash
Copy code
git rm -rf --cached .
Use a different network or VPN: If you're behind a corporate network or VPN, try disconnecting from it and using a different network connection.

Check the repository availability: Ensure that the remote repository you're trying to clone is available and accessible. You can try visiting the repository's URL in a web browser to verify its availability. For example: https://nomesfavoritos.com.br/nomes-para-free-fire/

Try a shallow clone: If the repository is large, you can try a shallow clone, which downloads only the most recent commits. This can help bypass potential issues with large repository sizes:

bash
Copy code
git clone --depth=1 https://github.com/substrate-developer-hub/substrate-node-template.git
Wait and retry: If the issue persists, you can wait for some time (e.g., an hour or more) and try cloning the repository again, as the problem might be related to a temporary issue with the remote server or network.

If none of these steps resolve the issue, it's possible that there might be a problem with the remote repository itself, or a more complex networking issue on your end. In such cases, you can try contacting the repository maintainers or seeking further assistance from the Git community or your network administrators.

**Reply by raggi** [reply]
@Bilalsheikh14 I realize you're trying to be helpful, but the high frequency issue here is, categorically, the Windows SSH port having a bug. As it's a race condition, if users retry enough they'll succeed, and as such a lot of claims of "it's your connection" _appear true_ because retries work.

The issue is windows SSH, the fix is in the unstable package. It can be fixed as commented here https://github.com/orgs/community/discussions/48568#discussioncomment-8510498 though as noted https://github.com/orgs/community/discussions/48568#discussioncomment-10068525 you need to ensure that the new ssh version is being used by git.

**Reply by MohsinSheeraz** [reply]
Thnx! 🙏🏻 works for me after executing with "git clone --depth=1"

**Reply by the-pro7** [reply]
Mine worked with `--depth=1`. Thanks pal!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @oliver-pyon ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you.

**Comment by HassanSj** [comment]
The error you're encountering, "fetch-pack: unexpected disconnect while reading sideband packet" along with "fatal: early EOF" and "fatal: fetch-pack: invalid index-pack output," often indicates a network or connectivity issue while cloning a Git repository from a remote server. Here are some steps to help you fix this issue:

1. **Check Your Internet Connection**: Ensure that your internet connection is stable and not experiencing interruptions. Slow or unstable internet connections can lead to incomplete downloads.

2. **Retry the Clone**: Sometimes, network issues are temporary. Retry the clone command to see if the issue resolves itself.

```shell
git clone https://github.com/substrate-developer-hub/substrate-node-template.git
```

3. **Use SSH**: If you continue to face issues with HTTPS cloning, consider using SSH to clone the repository. Ensure that you've set up an SSH key pair and added the public key to your GitHub account.

```shell
git clone git@github.com:substrate-developer-hub/substrate-node-template.git
```

4. **Increase Buffer Sizes**: You can try increasing Git's buffer sizes to allow for larger transfers. You can do this by setting the following configuration values:

```shell
git config --global http.postBuffer 524288000  # Set a larger buffer size
git config --global core.compression 0         # Disable compression
```

5. **Use a Git Client**: If the issue persists, consider using a Git client like GitKraken, Sourcetree, or GitHub Desktop, which often have built-in features to handle network issues more gracefully.

6. **Check GitHub Status**: Occasionally, GitHub itself may experience issues. You can check the GitHub Status page (https://www.githubstatus.com/) to see if there are any ongoing incidents that might be affecting repository access.

7. **Try Cloning Later**: If you suspect that the issue is on the GitHub server's side, it may be worth trying the clone operation at a different time.

8. **Contact GitHub Support**: If none of the above solutions work, and you continue to experience issues, consider reaching out to GitHub support for assistance with the specific repository or server-related problems.

9. **Check Proxy Settings**: If you are behind a corporate firewall or using a proxy server, ensure that your Git configuration reflects the correct proxy settings.

Keep in mind that the error message you're seeing is often related to network issues, so troubleshooting your network connectivity is a critical step in resolving this problem.

**Reply by isaacobenzy** [reply]
I am also having same but when i clone different repo's it works

**Reply by mdranacse19** [reply]
Thank you so much. you save me a lot of time.

**Reply by ravindrank** [reply]
> 4. **Increase Buffer Sizes**: You can try increasing Git's buffer sizes to allow for larger transfers. You can do this by setting the following configuration values:
> 
> ```shell
> git config --global http.postBuffer 524288000  # Set a larger buffer size
> git config --global core.compression 0         # Disable compression
> ```

I do have a slow internet and these settings helped!! Thanks

**Reply by MachineLearningMike** [reply]
Thank you. Use SSH works for me.

**Reply by madeehawaqas04** [reply]
in visual studio 2022 where write these

git config --global http.postBuffer 524288000  # Set a larger buffer size
git config --global core.compression 0         # Disable compression

**Comment by VDIAN01** [comment]
This is based on network issue. I also faced same issue while connecting mobile data So connected wifi issue is resolved.

**Reply by akashmuhammadabrrar** [reply]
I think im suffering the same issue what should i need to do for it. plz tell me there is no other way except my weak wifi network

**Comment by virajbhutada** [comment]


**Reply by queenofcorgis** [reply]
Hi @virajbhutada , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by watchpoints** [comment]
change https to ssh

**Reply by w0nd3r365** [reply]
Tried so many proposed solutions from stackoverflow, but this is the only one that worked when cloning github.com/jellyfin/jellyfin.org

**Reply by breck7** [reply]
This worked for me. Thank you.

**Comment by RaymanMartin** [comment]
I got the relativly issue like.

`Receiving objects: 100% (1103339/1103339), 29.19 GiB | 71.13 MiB/s, done.
error: inflate: data stream error (incorrect data check)
fatal: serious inflate inconsistency
fatal: fetch-pack: invalid index-pack output`

This git repository is about 30G，its a Aosp git repository

this issue raise persistently,  And i had try plently of method, but this issue still there.

**First**  : modify git config file.
`core.packedgitlimit=51200m
core.packedgitwindowsize=52100m
core.compression=0
pack.windowmemory=8096m
pack.packsizelimit=8096m
pack.deltacachesize=8096m
ssh.postbuffer=524288000
core.repositoryformatversion=0
core.filemode=true
core.bare=false
core.logallrefupdates=true
http.postbuffer=524288000`

didnt work for me

**Second**: acrroding to stack overflow，I had try modify the system property like ulimit file
`
real-time non-blocking time  (microseconds, -R) unlimited
core file size              (blocks, -c) 0
data seg size               (kbytes, -d) unlimited
scheduling priority                 (-e) 0
file size                   (blocks, -f) unlimited
pending signals                     (-i) 191275
max locked memory           (kbytes, -l) 6130312
max memory size             (kbytes, -m) unlimited
open files                          (-n) 2460000
pipe size                (512 bytes, -p) 8
POSIX message queues         (bytes, -q) 819200
real-time priority                  (-r) 0
stack size                  (kbytes, -s) 8192
cpu time                   (seconds, -t) unlimited
max user processes                  (-u) 191275
virtual memory              (kbytes, -v) unlimited
file locks                          (-x) unlimited
`
didnt work for me


**Third**: try to use git init and git remote add origin xxxurl, then git fetch --all --depath 1
didnt work for me

If theres any other solution？Or maybe i should change the network?

DeviceInfo:
Ubuntu：22.0.4
Git：2.34.1
Memory：48G ram and 50G swapFile size

**Reply by aliqramraju-git** [reply]
Same here. I have tried all the methods mentioned still having the problem.

**Comment by Kamellion-Dev** [comment]
mere network issue. connect to mobile data and you will O.K

**Reply by alhassan-cse** [reply]
good answer

**Reply by LynxTR** [reply]
sometimes you saw an answer and realise you overcomplicated things... thank you so much for the solution.

**Comment by sundayglee** [comment]
Use `git clone --recursive `

**Comment by akshaydhardubey** [comment]
you can use google colab or some other environment for git clone
then export the files using

 !tar -czf file_name.tar.gz {'file_path'}

replace file_path with the file path you want to export and file_name with the selective file name

**Comment by chrisdenton-ct** [comment]
I get this same issue on my iMac all the time. Cloning works fine outside of GitHub Desktop, and even GitHub Desktop works on a Macbook Air which is on the same WiFi.

**Comment by Hasan-Mir** [comment]
Had the same problem, I imported the repo from Github to Gitlab and cloned the project from Gitlab, which worked without any issues. 
On Github, the problem was resolved only after switching from WiFi to mobile network.

**Comment by SobanSiraj** [comment]
Git:  1683 bytes of body are still expected

**Comment by SobanSiraj** [comment]
no answer

**Comment by SobanSiraj** [comment]
github fetch issue

**Comment by 3iv4ns** [comment]
mine worked after restarting my laptop and switched to mobile connection

**Comment by zaheerahmad33** [comment]
I tried all of the above nothing worked for me

What worked for me

I updated the git bash to version 2.46.0 and issue solved!

**Comment by faroukcharkas** [comment]
I was getting this error while connected to an airport network, but then I turned on my VPN (NordVPN connected to Chicago) and I was able to download just fine.

**Comment by serge-salamanka** [comment]
fixed by updating GIT for Windows

**Comment by RafaelJohn9** [comment]
I had the same error when I had a slow network and when the GitHub repo was very bulky.
Changing to ssh type of connection fixed the error

**Comment by dcaayushd** [comment]
The error you're encountering, "error: 6546 bytes of body are still expected," and "fetch-pack: unexpected disconnect while reading sideband packet," typically indicates an issue with the network connection or the remote repository's availability.

Here are a few steps you can try to resolve the issue:

1. **Check your internet connection**: Make sure you have a stable internet connection. Try running a speed test or accessing other websites to verify that your connection is working properly.

2. **Retry the clone operation**: Sometimes, a temporary network glitch or server issue can cause the clone operation to fail. Try running the `git clone` command again.

3. **Use a different protocol**: Instead of using the HTTPS protocol (`https://github.com/...`), try cloning the repository using the SSH protocol (`git@github.com:...`). This can help bypass potential issues with HTTPS connections.

4. **Clear Git's cache**: Git stores a cache of remote repository data, which can sometimes become corrupted. Try clearing the cache by running the following command and then retrying the clone:

   ```
   git rm -rf --cached .
   ```

5. **Use a different network or VPN**: If you're behind a corporate network or VPN, try disconnecting from it and using a different network connection.

6. **Check the repository availability**: Ensure that the remote repository you're trying to clone is available and accessible. You can try visiting the repository's URL in a web browser to verify its availability.

7. **Try a shallow clone**: If the repository is large, you can try a shallow clone, which downloads only the most recent commits. This can help bypass potential issues with large repository sizes:

   ```
   git clone --depth=1 https://github.com/substrate-developer-hub/substrate-node-template.git
   ```

8. **Wait and retry**: If the issue persists, you can wait for some time (e.g., an hour or more) and try cloning the repository again, as the problem might be related to a temporary issue with the remote server or network.

If none of these steps resolve the issue, it's possible that there might be a problem with the remote repository itself, or a more complex networking issue on your end. In such cases, you can try contacting the repository maintainers or seeking further assistance from the Git community or your network administrators.

**Comment by IndevGames** [comment]
Write the bellow command before cloning. this worked for me.

git config --global http.postBuffer 1024M

**Comment by Micheal5699** [comment]
All the above solutions didn't work until I switched to my mobile data before it worked.
Switch to your Mobile Data and re-run it, it will work 👍🏽✅

**Comment by Velcent** [comment]
git is very bad for cloning repos
I use github desktop and never encounter any errors

**Comment by syedauroosa11** [comment]
Question:
During bitbake i run this command:
uroosa@uroosa27:~/yocto/build$ bitbake linux-xlnx
but i get this error
WARNING: linux-xlnx-6.6.10+git-r0 do_fetch: Failed to fetch URL git://github.com/Xilinx/linux-xlnx.git;protocol=https;name=machine;branch=xlnx_rebase_v6.6_LTS, attempting MIRRORS if available
ERROR: linux-xlnx-6.6.10+git-r0 do_fetch: Fetcher failure: Fetch command export PSEUDO_DISABLED=1; export DBUS_SESSION_BUS_ADDRESS="unix:path=/run/user/1000/bus"; export SSH_AUTH_SOCK="/run/user/1000/keyring/ssh"; export PATH="/home/uroosa/yocto/build/tmp/sysroots-uninative/x86_64-linux/usr/bin:/home/uroosa/yocto/sources/poky/scripts:/home/uroosa/yocto/build/tmp/work/zybo_zynq7-poky-linux-gnueabi/linux-xlnx/6.6.10+git/recipe-sysroot-native/usr/bin/arm-poky-linux-gnueabi:/home/uroosa/yocto/build/tmp/work/zybo_zynq7-poky-linux-gnueabi/linux-xlnx/6.6.10+git/recipe-sysroot/usr/bin/crossscripts:/home/uroosa/yocto/build/tmp/work/zybo_zynq7-poky-linux-gnueabi/linux-xlnx/6.6.10+git/recipe-sysroot-native/usr/sbin:/home/uroosa/yocto/build/tmp/work/zybo_zynq7-poky-linux-gnueabi/linux-xlnx/6.6.10+git/recipe-sysroot-native/usr/bin:/home/uroosa/yocto/build/tmp/work/zybo_zynq7-poky-linux-gnueabi/linux-xlnx/6.6.10+git/recipe-sysroot-native/sbin:/home/uroosa/yocto/build/tmp/work/zybo_zynq7-poky-linux-gnueabi/linux-xlnx/6.6.10+git/recipe-sysroot-native/bin:/home/uroosa/yocto/sources/poky/bitbake/bin:/home/uroosa/yocto/build/tmp/hosttools"; export HOME="/home/uroosa"; LANG=C git -c gc.autoDetach=false -c core.pager=cat -c safe.bareRepository=all clone --bare --mirror https://github.com/Xilinx/linux-xlnx.git /home/uroosa/yocto/build/downloads/git2/github.com.Xilinx.linux-xlnx.git --progress failed with exit code 128, see logfile for output
ERROR: linux-xlnx-6.6.10+git-r0 do_fetch: Bitbake Fetcher Error: FetchError('Unable to fetch URL from any source.', 'git://github.com/Xilinx/linux-xlnx.git;protocol=https;name=machine;branch=xlnx_rebase_v6.6_LTS')
ERROR: Logfile of failure stored in: /home/uroosa/yocto/build/tmp/work/zybo_zynq7-poky-linux-gnueabi/linux-xlnx/6.6.10+git/temp/log.do_fetch.109576
ERROR: Task (/home/uroosa/yocto/sources/meta-xilinx/meta-xilinx-core/recipes-kernel/linux/linux-xlnx_6.6.10-v2024.1.bb:do_fetch) failed with exit code '1'


Then i tried to download it manually using command: 
git clone git@github.com:Xilinx/linux-xlnx.git


after successful cloning i added the cloned repository to the specified path but still i am gettin g the same issue why? Please help me in this regard i am trying to do this from more than 3 days.

**Comment by aouiniamine** [comment]
What worked for me was connecting with SSH and then did:
```bash
git clone git@github.com:<organization>/<repository>.git
```

**Comment by diixo** [comment]
Git-2.47 has this issue (windows-version). I rolled-back to Git-2.42 and issue was fixed by this way. Looks like a really bug of newest Git.

**Reply by snigdhasjg** [reply]
Thanks for saving my hours. It has been fixed 3 hours back, but you have to download [v2.47.0.windows.2](https://github.com/git-for-windows/git/releases/tag/v2.47.0.windows.2) from github

**Comment by egozavr** [comment]
The only solution that worked for me was to uncheck "use OpenSSH bundled with git " and to choose "Use external OpenSSH" instead (windows native one). OS windows 11, git version 2.47.0.

**Comment by goursay** [comment]
I might be late to the party, but using git commands via WSL worked for me.

**Comment by snigdhasjg** [comment]
with latest version the bug is fixed but download from github not from website (yet)

https://github.com/git-for-windows/git/releases/tag/v2.47.0.windows.2

**Comment by frgthyjuierjk** [comment]
Just updated to `git version 2.47.0.windows.2` and the problem still persists. If I use `git fetch --unshallow` and the repository is somewhat big in size, it will still give a fatal error:

```bash
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
```

It only worked on **very small repositories**.

**Comment by baxteran** [comment]
A bit left-field, but I had exactly this problem and the solution for me was to follow the steps here - https://answers.microsoft.com/en-us/windows/forum/all/low-internet-speed-in-wsl-2/21524829-18be-4611-bb5f-cabccd2cae31
Turned out to be a problem with WSL network config. Hope this is a help to someone.

**Comment by souvik666** [comment]
guys if nothing works just use  `gh`

**Comment by wmeints** [comment]
I was running into this problem a lot lately, and I figured that the settings proposed here and elsewhere would help. Sadly that's not the case. But I figured it out. My router was running outdated firmware and misbehaving in other situations as well as when using SSH. But only in Linux. I updated the firmware, gave the router a reboot and everything is now running awesome again.

**Comment by Jay-study-nildana** [comment]
Using --depth=1 worked for me.

**Reply by Jay-study-nildana** [reply]
And it only seems to happen with some specific repos of mine, but rest of the repos work just fine even when cloned at the same time. 

(So, it cannot be a network or a computer issues)
